File: 백준/Bronze/28691. 정보보호학부 동아리 소개/정보보호학부 동아리 소개.cc
#include <stdio.h>

int main(void) {
	char question;
	scanf("%c", &question);
	switch (question) {
	case 'M':
		printf("MatKor");
		break;
	case 'W':
		printf("WiCys");
		break;
	case 'C':
		printf("CyKor");
		break;
	case 'A':
		printf("AlKor");
		break;
	case '$':
		printf("$clear");
		break;
	}
	return 0;
}

File: 백준/Bronze/28691. 정보보호학부 동아리 소개/정보보호학부 동아리 소개.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String code = br.readLine();
        switch(code.charAt(0)) {
            case 'M':
                System.out.println("MatKor");
                break;
            case 'W':
                System.out.println("WiCys");
                break;
            case 'C':
                System.out.println("CyKor");
                break;
            case 'A':
                System.out.println("AlKor");
                break;
            case '$':
                System.out.println("$clear");
                break;
        }
    }
}